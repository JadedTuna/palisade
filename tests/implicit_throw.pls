in {
    high secret: bool;
}
out {
    low secretSteal: bool;
}

secretSteal = false;
try {
    if (secret) {
        throw;
    }
} catch {
    secretSteal = true;
}
