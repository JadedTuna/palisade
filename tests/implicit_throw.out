[1;35msecurity error: [0mthrow in high context is not allowed
   9 | try {
  10 |     if (secret) {
  11 |         [1;35mthrow[0m;
       [1;35m~~~~~~~~[0m[1;35m^^^^^[0m
